### PR TITLE
feat: send sms receipt

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
[Offer SMS receipt](https://trello.com/c/WS8yKiGu/906-offer-sms-receipt)

File: lib/brain.js [hunk 1]
```diff
@@ -2252,7 +2257,7 @@ Brain.prototype._start = function _start () {
 
   this.browser().send({
     tx: this.tx,
-    receiptStatus: this.trader.receiptPrintingActive ? 'available' : 'disabled'
+    receiptStatus: this.trader.receiptPrintingActive ? 'available' : 'disabled',
```
Comment (chaotixkilla) [inline lib/brain.js:2260]: Any reason for this change? Only thing I could think of is the prettier config with `trailingComma: true`

Comment (chaotixkilla) [reply lib/brain.js:2260]: Or, thinking better about this change, seems as a leftover change where you added a field that was later deleted during development

File: lib/brain.js [hunk 2]
```diff
@@ -3369,7 +3410,7 @@ Brain.prototype._chooseFiat = function _chooseFiat () {
 
   this._transitionState('chooseFiat', {
     chooseFiat: response,
-    receiptStatus: this.trader.receiptPrintingActive ? 'available' : 'disabled'
+    receiptStatus: this.trader.receiptPrintingActive ? 'available' : 'disabled',
```
Comment (chaotixkilla) [inline lib/brain.js:3413]: Same here

File: lib/trader.js [hunk 3]
```diff
@@ -317,6 +317,15 @@ Trader.prototype.triggerSuspend = function triggerSuspend (customerId, triggerId
     .then(r => r.body)
 }
 
+Trader.prototype.smsReceipt = function smsReceipt (data, customerId) {
+  return this.request({
+    path: `/customer/${customerId}/smsreceipt`,
+    body: { data },
```
Comment (chaotixkilla) [inline lib/trader.js:323]: Isn't `data` already an object?

Comment (chaotixkilla) [reply lib/trader.js:323]: You're accessing this with `req.body.data` on server-side, due to this. Changing this has little to no impact, just pointed it out due to other functions in the trader module sending the object as is, without encapsulating on a `data` field

**Review by chaotixkilla — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**